### ISSUE TITLE
[Feature Request] Sails.js projects

### ISSUE THREAD

**Issue by dunkno** [balderdashy/www.sailsjs.org#73]
Wasn't too sure where the best place this issue would be, so I felt it would be most relevant here.

I've seen quite a few projects that use Sails now (I have a few too) and thought it would be a great idea to have a section on the website which demonstrates these projects.

Something like the [Built with Angular.js](http://builtwith.angularjs.org/) website would be brilliant!

**Comment by rachaelshaw** [comment]
@mwils-bp I like that idea. Right now all we have is [this list](https://github.com/balderdashy/sails-docs/blob/master/FAQ.md#who-else-is-using-sailsjs), but it's difficult to find and therefore pretty out-of-date.

I think the best way to approach this would be to make a new file in [sails-docs](https://github.com/balderdashy/sails-docs) so we can compile it the same way as the Get Started/Docs/Version Notes pages. We could create a new folder in that repo, and have each project be its own .md file in that folder. (It might be a lot of very short files, but it can all end up on the same page on the site.) I can make an example template file that people can copy, to keep the formatting the same.

**Comment by rachaelshaw** [comment]
Started setting up a page for this. Also [made a folder in sails-docs](https://github.com/balderdashy/sails-docs/tree/master/projects) for projects. (Still need to sort out some details w/ exactly what information we want people to submit and such, but should be ready to to start collecting project info for this soon.)

**Comment by sailsbot** [comment]
Thanks for posting, @dunkno.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
